app/components/support_interface/validation_errors_summary_component.html.erb

```diff
@@ -15,10 +15,9 @@
                 options: {
                   selected: params['sortby'] || 'all_time',
                 },
-                class: 'sortedby-label',
-                selected: 2,
-                onchange: 'this.form.submit()',
-                role: 'listbox',
```

Comment:
Why do we need to get rid of `class`, `selected` and `role` here?

Reply:
I've now added these back in